app/views/errors/not_found.html.erb

```diff
@@ -0,0 +1,30 @@
+<% set_meta_tags(title: t("errors.show.#{@rescue_response}.title")) %>
+<%= heading_tag t("errors.show.#{@rescue_response}.heading", default: t('errors.show.heading')), class: "error-page__headline" %>
+
+<p class="error-page__text<% if @rescue_response != :not_found %> error-page__text--only<% end %>"><%= t("errors.show.#{@rescue_response}.intro", default: t("errors.show.#{@rescue_response}.title")) %></p>
```

Comment:
well spotted. This page was a generic page. I will take out the status code checks.